tests/test_routes.py

```diff
@@ -18,7 +18,8 @@
     "DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/postgres"
 )
 
-BASE_URL = "/wishlists"
+OLD_BASE_URL = "/wishlists"
```

Comment:
We should stay away from naming things "old" or "new" ever because those things are temporary.  This would be better as "UI_BASE_URL" and "API_BASE_URL", but it's 1:15am the night before this is due so we'll let it slide! lol

Reply:
@jay-goettelmann-nyu yeah youre totally right, should have dealt with it the other way!